protocol/messenger_handler.go

```diff
@@ -213,6 +215,16 @@ func (m *Messenger) HandleMembershipUpdate(messageState *ReceivedMessageState, c
 	return nil
 }
 
+func (m *Messenger) checkIfCreatorIsOurContact(group *v1protocol.Group) bool {
+	creator, err := group.Creator()
+	if err == nil {
+		contact, _ := m.allContacts.Load(creator)
+		return contact != nil && contact.Added && contact.HasAddedUs
```

Comment:
I think you should be able to use `ContactRequestStateMutual` in `ContactRequestState`, it should be the same, and we will be moving away from `Added` `HasAddedUs`

Reply:
well, i think we can't use ContactRequestState here, i've tested with:
```
if contact.Added && contact.HasAddedUs && contact.ContactRequestState != ContactRequestStateMutual {
	println("bomb!")
}
```
we will c "bomb!" @cammellos

Reply:
will fix the inconsistent in a separate PR